### ISSUE TITLE
Proposal: Seqkit stats when demultiplexing

### ISSUE THREAD

**Issue by a4000** [nf-core/ampliseq#611]
### Description of feature

I know Ampliseq already uses fastqc, but I like using Seqkit stats when demultiplexing to track the number of reads that did and didn't get assigned to samples. There is another issue proposing a demultiplex step that I hope to resolve, so I figured I'd propose Seqkit stats for when the user chooses to demultiplex.

**Comment by a4000** [comment]
I agree on the subworklow. There are other modules I might need to add to the subworkflow to get around a problem I've identified with demultiplexing. I'll make another issue to go into more details about that problem, and my proposed solution

**Comment by d4straub** [comment]
I close that as "not planned" for now, but please anyone re-open this in case someone wants to take action here.
